tests/test_anisotropic_noise.py

```diff
@@ -16,7 +16,7 @@ class SimTestCase(TestCase):
     def setUp(self):
         self.dtype = np.float32
         self.vol = LegacyVolume(L=8, dtype=self.dtype).generate()
-        self.sim = Simulation(
+        self.sim = LegacySimulation(
```

Comment:
So am I understanding things correctly in that `LegacySimulation` is only needed in places where we test against hardcoded reference values? So if (when?) we get rid of those, we should also be able to rid ourselves of `LegacySimulation`?

Reply:
That is correct.

Reply:
Got it.